client/src/components/Monument/Monument.js

```diff
@@ -1,256 +1,40 @@
-                {id: 4, url: 'https://lh5.googleusercontent.com/p/AF1QipOwnKkvd1BHSv_I8FetfXLT7q01w1n6e3xPmzbn=w203-h270-k-no'},
-                {id: 5, url: 'https://lh5.googleusercontent.com/p/AF1QipOvJE2czQBHI9rmkIXNqM8AKA6kZSxV8DpAN1Xr=s1016-k-no'},
-                {id: 6, url: 'https://lh5.googleusercontent.com/p/AF1QipOwnKkvd1BHSv_I8FetfXLT7q01w1n6e3xPmzbn=w203-h270-k-no'}
-            ], references, tags});
-        console.log(this.state);
-    }
-
-    async callEndpoint(endpoint) {
-        const response = await fetch(endpoint);
-        const parsed = await response.json();
-        if (parsed.errors) {
-            throw(parsed.errors[0]);
-        }
-        return parsed;
-    }
-
-    /**
-     * This function encapsulates the logic to add the slug at the end of the url if it's not present
-     */
-    redirectToSlug() {
-        const monument = this.state.monument;
-        const slug = this.state.slug;
-        // Wait for the monument to be loaded in from the API
-        // If there's no title, slugify will throw an error, so only proceed if there's a title
-        if (!monument || !monument.title) return;
-        // Slugify the monument's title
-        const newSlug = slugify(monument.title);
-        // Don't redirect if the correct slug is already present
-        if (slug !== newSlug) {
-            // Redirect via React Router
-            return <Redirect to={`/monuments/${monument.id}/${newSlug}`}/>;
-        }
-    }
-
-    prettyPrintDate(date) {
-        if (!date) return;
-        date = moment(new Date(date));
-        // Wednesday, October 16th, 2019 format
-        return date.format('dddd, MMMM Do, YYYY');
-    }
-
-
-    capitalize(word) {
-        if (word) return word.charAt(0).toUpperCase() + word.slice(1).trim();
-        else return word;
-    }
-
-    parseState(state) {
-        if (!state) return state;
-        if (state.toLowerCase() === 'dc') state = 'd.c.';
-        return state.toUpperCase().trim();
-    }
-
-    formatInscription(inscription) {
-        if (!inscription) return inscription;
-        // This nifty little regex removes all of the double quotes in the string
-        inscription = inscription.replace(/["]+/g, '');
-        return '"' + inscription + '"';
-    }
-
     render() {
-        if (this.state.error) return this.renderError();
-
-        const title = this.state.monument.title;
+        const { monument } = this.props;
+        if (!monument) return (<div/>);
+        const title = monument.title;
 
         return (
             <div className="page-container">
                 <Helmet title={title + ' | Monuments and Memorials'}/>
-                {this.redirectToSlug()}
                 <div className="column related-monuments-column">
-                    {this.renderSuggestChanges()}
+                    <SuggestChanges/>
                     {this.renderNearbyMonuments()}
                     {this.renderRelatedMonuments()}
```

Comment:
I'm guessing we'll split these into their own Components when we implement them?

Reply:
Lol nevermind just saw the comment a few lines down.